.evergreen/config.yml

```
@@ -586,7 +584,8 @@ tasks:
           ./bin/pip install poster
           ./bin/pip install pycrypto
           cd ..
-          pkg_version=$mongocrypt_version
+          # Get the current version of libmongocrypt.
+          pkg_version="$(python etc/calc_release_version.py 2>&1)"
```

```suggestion
          pkg_version="$(python etc/calc_release_version.py)"
```

Running with python 3.10.6 shows the following deprecation warnings:
```
python ./etc/calc_release_versionNoGit.py 
/Users/kevin.albertson/review/libmongocrypt-446/./etc/calc_release_versionNoGit.py:214: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  version_loose = LooseVersion('0.0.0')
/Users/kevin.albertson/review/libmongocrypt-446/./etc/calc_release_versionNoGit.py:73: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  version_loose = LooseVersion('0.0.0')
/Users/kevin.albertson/review/libmongocrypt-446/./etc/calc_release_versionNoGit.py:86: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  new_version_loose = LooseVersion(release_tag_match.group('ver'))
1.5.0-alpha0
```

Redirecting stderr to stdout may cause deprecation warnings to be captured.

IMO this does not need to be resolved in this PR. But this may require implementing an alternative to `LooseVersion` in the future.

---

@kevinAlbs I concur that we may need to think about alternatives to `LooseVersion`.  I ran into the warnings as well recently when working on syncing between the C and C++ driver.